src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Person's birthday in the address book.
+ * Guarantees: immutable; is always valid
+ */
+public class Birthday {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Birthday should be of the format yyyy-mm-dd";
+    public static final String VALIDATION_REGEX = "^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[12]\\d|3[01])$";
```

Comment:
Good try but this only examine the literals format and misses some sematic checks as mentioned earlier

Reply:
Okie, removed and changed to purely LocalDate